pkg/features/kube_features.go

```diff
@@ -178,6 +178,14 @@ const (
 	// Enables kubelet to detect CSI volume condition and send the event of the abnormal volume to the corresponding pod that is using it.
 	CSIVolumeHealth featuregate.Feature = "CSIVolumeHealth"
 
+	// owner: @seans3
+	// kep: http://kep.k8s.io/4006
+	// alpha: v1.29
+	//
+	// Enables StreamTranslator proxy to handle WebSockets RemoteCommand/V5
+	// upgrade requests.
+	ClientRemoteCommandWebsockets featuregate.Feature = "ClientRemoteCommandWebsockets"
```

Comment:
This gate name is sort of confusing, since it controls server behavior, not client. also, will we use the same gate for portforward? that's not remotecommand, right?

Maybe `TranslateStreamCloseWebsocketRequests` or something similar would be closer to what this is actually controlling?

Reply:
Changed feature gate name to `TranslateStreamCloseWebsocketRequests`.